components/calendar/src/islamic.rs

```diff
@@ -1465,4 +1463,41 @@ mod test {
             );
         }
     }
+
+    #[ignore]
+    #[test]
+    fn test_days_in_provided_year_observational() {
+        // -1245 1 1 = -214526 (R.D Date)
+        // 1518 1 1 = 764589 (R.D Date)
+        let x_year = -1245;
+        let y_year = 1518;
+
+        let sum_days_in_year: i32 = (x_year..y_year)
+            .map(|year| IslamicObservational::days_in_provided_year(year) as i32)
+            .sum();
+        let expected_number_of_days = 979115; // The number of days between Islamic years -1245 and 1518
```

Comment:
Nit, here and below: Test the code that generates this number.

Reply:
done.